packages/server/beaver_triple_service/jwt/types.go

```diff
@@ -3,7 +3,7 @@ package jwt
 import "github.com/golang-jwt/jwt/v4"
 
 type Party struct {
-	Id      int    `json:"id"`
+	Id      uint32    `json:"id"`
 	Address string `json:"address"`
```

Comment:
It's a very small detail, but I would appreciate it if you could align the columns of the struct tags.

```suggestion
	Id      uint32 `json:"id"`
	Address string `json:"address"`
```

Reply:
Corrected with [a8f97e7](https://github.com/acompany-develop/QuickMPC/pull/122/commits/a8f97e7d47f513efa7b99aa3cad34ecfcca0185a)